MetaData/work/campaigns/RunIIFall17-3_0_0_part5.json

```diff
@@ -0,0 +1,9 @@
+{
+	"data" : [
+	       ],
+	"sig"  : [
+	       ],
+	"bkg"  : [
+"/DYJetsToLL_M-50_TuneCP5_13TeV-amcatnloFXFX-pythia8/RunIIFall17MiniAOD-94X_mc2017_realistic_v10-v1/MINIAODSIM"
```

Comment:
@sethzenz  Just to mention there are also large DY samples per jet bins; currently the 3 jet[ bin](https://cms-pdmv.cern.ch/pmp/historical?r=HIG-RunIIFall17MiniAOD-00826) is missing though

Reply:
@sethzenz 
Here the samples on DAS.https://cmsweb.cern.ch/das/request?view=plain&limit=50&instance=prod%2Fglobal&input=dataset%3D%2FDY*JetsToLL_M-50_TuneCP5_13TeV-madgraphMLM-pythia8%2FRunIIFall17MiniAOD-94X_mc2017_realistic_v10-v1%2FMINIAODSIM

The missing sample has been submitted, but it is simply in the production queue

Reply:
Thanks.  Indeed the missing sample doesn't exist yet as MINIAODSIM in any status.  i.e. this returns only the samples 1,2, and 4:

dasgoclient -query="dataset dataset=/DY*JetsToLL_M-50_TuneCP5_13TeV-madgraphMLM-pythia8/RunIIFall17MiniAOD-94X_mc2017_realistic_v10-v1/MINIAODSIM status=*"

Given that i'm not sure we need to process any of them just yet, since this version of the flashgg recipe needs some validation anyway.  But let me know what you think!

Reply:
@sethzenz Sure no problem, I just pointed it out in case they were missed.